### ISSUE TITLE
Flying Camera

### ISSUE THREAD

**Issue by ghost** [PlaceholderGames/2016-VnD_Game#58]
As discussed in class the camera can be improved on by changing it to a flying first person camera and removing the attached character model.  I feel this will allow the player to navigate throughout the level with ease. Also, the ability to see the level from an aerial view allows the player to make more informed choices.

**Comment by ghost** [comment]
I’ve implemented a solution for this which keeps the camera locked to the floor, removes the character model, walking animations and camera movement.  I’d like to show you guys in class Thursday if your happy with the implementation we can merge it.

**Comment by BojanStankovic** [comment]
You can work on it in this issue's branch and push it for the review. No need to wait for Thursday.

**Comment by ghost** [comment]
Yes, I realised that was unnecessary.  I’ve pushed the solution to this issues branch, this issue is now up for review, leave me some feedback.

**Comment by napiorek** [comment]
Just tested it, it's  flying on x and y axis but I could not go up ? (z axis) 
And if you look at the map, the camera is still attached to the 3rd person.

![58_camera](https://cloud.githubusercontent.com/assets/22525895/23622827/0a889630-0298-11e7-8be0-d16dd2280a1d.JPG)

**Comment by rbakerusw** [comment]
Is there any bounds checking to prevent the camera from going too far from the level?

**Comment by BojanStankovic** [comment]
I removed the "review me!" tag since this is still WIP. Those two tags shouldn't be valid for an issue at the same time

**Comment by napiorek** [comment]
Hi @Richard-Chaos, can You update us on the progress of this issue ?
As currently @BojanStankovic discover the bug that involve the 3rd person character. 
It's to do with ending condition(flag) so during a game, if 3rd person character (that camera is attached to) will step on this flag it freeze the 3rd person character in this position and you can not move the camera any more.

It not gonna be an issue once we detached the 3rd person from camera.

**Comment by ghost** [comment]
Hey guys, I've updated the camera its now working as it should be. Can you test it to see if it feels right to you? It feels alot easier to play the game now.

I'm creating a new issue for the level boundaries now.
